docs/src/gui/qtvcp-vcp-panels.adoc

```diff
@@ -244,22 +257,9 @@ net s32-in1        test_panel.doublescale_1-s   classicladder.0.s32in-00
 start
 ----
 
-In this case we load qtvcp using -Wn; which waits for the panel to
-finish loading before continuing to run the next HAL command. This is so
-the HAL pins from the panel are finished in case the are used in the
-rest of the file.
-
-
-== More Information
-
-<<cha:qtvcp-widgets,QtVCP Widgets>>
-
-<<cha:qtvcp-libraries,QtVCP Libraries>>
-
-<<cha:qtvcp-code,QtVCP Handler File Code Snippets>>
-
-<<cha:qtvcp-development,QtVCP Development>>
-
-<<cha:qtvcp-custom-widgets,QtVCP Custom Designer Widgets>>
```

Comment:
Why not keeping these references?

Reply:
I admit to have wondered, too.

Reply:
Because this is already present at the bottom of the qtvcp.adoc file. If this makes sense for the latter as a "QtVCP Guide" ToC, I don't see the point of putting it here again. Else we'd have to put it at the end of each chapter which doesn't really make sense and which we don't do anywhere else.

Reply:
Reduced redundancies are fine with me.

Reply:
I saw it more as a cross reference as we have it in several man pages like @petterreinholdtsen was recently improving in e.g. https://github.com/LinuxCNC/linuxcnc/pull/2016.
But it's up to how you would like to structure it, just wanted to hear your thoughts 😉

Reply:
I get the cross reference point and support improvements such as @petterreinholdtsen 's, but I feel they happen at a more appropriate finer grain level. Cross-referencing is a field of improvement but IMHO should be done mostly inline and at the most accurate level possible.

If really desirable we could replace this with a cross-ref to the QtVCP guide ToC... But wouldn't it be better provided with automatic Back/Up/Home/Next links ?

Reply:
Or add something like "for an overview of QtVCP related stuff see ...../QtVCP.html".
But this should be clear so don't have to be mentioned here.